Session1/example10_user_input/example10_user_input.py

```diff
@@ -0,0 +1,15 @@
+# !/usr/bin/env python3
+# @file example10_user_input.py
+# SCRP: Example 10 - User Input
+# Daryl Dang
+
+# Basic string example
+message_from_user = input("Type your message here: ")
+print("This is what the user said: " + message_from_user)
+
+# Integer example
+number_from_user = input("Enter in a number from 1 to 5: ")
```

Comment:
`number_from_user = 0 #Initialized as zero to start prompt`
`while(number_from_user < 1 or number_from_user > 5): #checks if value is not between 1 and 5`
`number_from_user = input("Enter in a number from 1 to 5: ")` 
^if you want to add some if statements into the exercise